src/main/java/gov/nasa/jpl/mbee/mdk/api/incubating/MDKConstants.java

```diff
@@ -44,9 +44,9 @@
             ORG_ID_KEY = ORG_KEY + ID_KEY_SUFFIX,
             TWC_ID_KEY = "twc" + ID_KEY_SUFFIX,
             TWC_VERSION_KEY = "twcVersion",
-            ARTIFACT_IDS_KEY = "_artifactIds",
+            ARTIFACTS_KEY = "_artifacts",
             CHECKSUM_KEY = "checksum",
-            CONTENT_TYPE_KEY = "contentType",
+            CONTENT_TYPE_KEY = "mimetype",
```

Comment:
Should define a new constant instead of redefining an existing one, and optionally deprecating this one if there are no longer any MDK-internal uses.